src/components/CreateMatch/components/CompetitorAlert/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { firebaseConnect } from 'react-redux-firebase';
+import PropTypes from 'prop-types';
+import { BounceIn } from 'animate-css-styled-components';
+import {
+  AlertWrapper,
+  WinnerWarningCenter,
+  BoomText,
+  BoomTextShadow,
+} from './styles';
+import Avatar from './../../../shared/components/Avatar';
+
+const CompetitorAlert = props => {
+  const calculateStreak = gameList => {
+    let streak = 1;
+    gameList.split('').reduceRight((prev, current) => {
+      if (current === prev) {
+        streak++;
+      }
+      return prev;
+    });
+    return streak;
+  };
+
+  const setVictoryName = streak => {
+    if (streak >= 10) {
+      return 'godlike';
+    } else if (streak >= 6) {
+      return 'rampage';
```

Comment:
![image](https://user-images.githubusercontent.com/1035101/37968639-050c18a0-31cf-11e8-8fba-daf18f22b442.png)

Reply:
laugh all you want...its gonna be awesome